app/services/licences/fetch-licence-has-requirements.service.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+/**
+ * Fetches requirements for a licence and returns and returns a boolean if the licence has requirements for returns
+ * @module FetchLicenceHasRequirementsService
+ */
+
+const ReturnVersionModel = require('../../models/return-version.model.js')
+
+/**
+ * Fetches requirements for a licence and returns a boolean if the licence has requirements for returns
+ *
+ * @param {string} licenceId - The UUID for the licence id to fetch
+ *
+ * @returns {Promise<Boolean>} the result of check if a licence has requirements for returns
+ */
+async function go (licenceId) {
+  const requirement = await _fetch(licenceId)
+
+  return !!requirement
+}
+
+async function _fetch (licenceId) {
+  return ReturnVersionModel.query()
+    .select([
+      'id'
+    ])
+    .where('licenceId', licenceId)
+    .first()
```

Comment:
[Objection.js first()](https://vincit.github.io/objection.js/api/query-builder/other-methods.html#first) doesn't actually limit the number of results from the query, it is just a helper to return the first one.

In these cases, I find it more explicit to highlight we only want one result by including a [limit()](https://vincit.github.io/objection.js/api/query-builder/find-methods.html#limit).

```suggestion
  return ReturnVersionModel.query()
    .select([
      'id'
    ])
    .where('licenceId', licenceId)
    .limit(1)
    .first()
```